tools/bin/go_core_tests

```diff
@@ -2,10 +2,13 @@
 set -o pipefail
 set +e
 
+SCRIPT_PATH=`dirname "$0"`; SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
+OUTPUT_FILE="./output.txt"
+
 echo "Failed tests and panics: ---------------------"
 echo ""
 GO_LDFLAGS=$(bash tools/bin/ldflags)
-go test -ldflags "$GO_LDFLAGS" -tags integration -p 3 -coverprofile=coverage.txt -covermode=atomic ./... | tee ./output.txt | grep --line-buffered --line-number -e "\-\-\- FAIL" -e "FAIL\s"
+go test -ldflags "$GO_LDFLAGS" -tags integration -p 3 -coverprofile=coverage.txt -covermode=atomic ./... -v | tee $OUTPUT_FILE | grep --line-buffered --line-number -e "\-\-\- FAIL" -e "FAIL\s"
```

Comment:
Hmmm why revive the `-v`? It was just removed recently since all the tests are finally scoped through their own test loggers: #6742

Reply:
Ah you're right, I tested and it works as expected without the `-v`, updated 🙂